test/e2e_node/cpu_manager_test.go

```diff
@@ -417,6 +436,64 @@ func runCPUManagerTests(f *framework.Framework) {
 
 		setOldKubeletConfig(f, oldCfg)
 	})
+
+	It("should reassigne CPUs after kubelet restart", func() {
```

Comment:
Is it necessary to have a separate `It` spec ? Can we use the same one ?

Reply:
It was pretty reasonable for me to split it into another test :D

Reply:
Please move this to the existing `It`. It will also remove most of the surrounding boiler plate code.